code/__HELPERS/unsorted.dm

```diff
@@ -1904,8 +1904,7 @@ var/mob/dview/dview_mob = new
 
 	for(var/mob/M in mobs)
 		if(skip_mindless && (!M.mind && !M.ckey))
-			if(!isbot(M) && !istype(M, /mob/camera/))
```

Comment:
Doesn't make sense to me so I removed it.

Reply:
Presumably, this is so that ghosts could follow beepsky or medbots or similar. It also allows ghosts to follow the AI's camera from that list, which doesn't normally have a client directly attached to it